src/browser/api_protocol/api_handlers/system.ts

```diff
@@ -556,7 +556,7 @@ function resolveUuid(identity: Identity, message: APIMessage, ack: Acker, nack:
 
 function getPreloadScripts(identity: Identity, message: APIMessage, ack: Acker, nack: Nacker): void {
     System.getPreloadScripts(identity)
-        .then((preloadScripts) => {
+        .then((preloadScripts: any) => {
```

Comment:
Please, use preload script's shape we already have here: https://github.com/HadoukenIO/core/blob/develop/src/shapes.ts#L292